### PR TITLE
Headphone Landing Page

### PR DESCRIPTION
**Pull request by itzwow**
# Description

Added a new Template "Headphone Landing Page"

Fixes:  #861 

## Type of change

-
- [x ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ x] I have made this from my own 
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
preview

### CODE REVIEW

File: Headphone-Landing-Page/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (VidhiBhatt01) [inline Headphone-Landing-Page/index.html:8]: Add proper title

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
make it responsive and remove extra white space from the side
image

**Comment by VidhiBhatt01** [comment]
https://deploy-preview-866--cool-front-end-templates.netlify.app/headphone-landing-page/

**Comment by itzwow** [comment]
I have updated the code and made it responsive



No more white space in the side 
Changed the title of the page:

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Footer alignment in not proper and now some white space is added at the end
image

**Comment by itzwow** [comment]
> Footer alignment in not proper and now some white space is added at the end image

Actually the social icons are designed such way only to keep it at a fixed position.

The color issue has been resolved
is
d

**Review by DEEPIKA-SUK — APPROVED** [review]

**Review by arpit456jain — APPROVED** [review]

**Pull request merged**